engineering_tools/carma_script_extensions/swarm

```diff
@@ -261,23 +271,22 @@ swarm__register() {
     ssh -R "$REGISTRY_PORT:$REGISTRY_HOST:$REGISTRY_PORT" "$WORKER_INFO" sh
     for TARGET_IMAGE_NAME in $IMAGE_NAMES
     do
-        echo "Pulling \$TARGET_IMAGE_NAME onto $WORKER_INFO from $REGISTRY_HOST:$REGISTRY_PORT"
-        docker pull "$REGISTRY_HOST:$REGISTRY_PORT/\$TARGET_IMAGE_NAME" \
-          && docker tag "$REGISTRY_HOST:$REGISTRY_PORT/\$TARGET_IMAGE_NAME" "\$TARGET_IMAGE_NAME" \
-          && docker rmi "$REGISTRY_HOST:$REGISTRY_PORT/\$TARGET_IMAGE_NAME"
+        echo "Pulling $TARGET_IMAGE_NAME onto $WORKER_INFO from $REGISTRY_HOST:$REGISTRY_PORT"
+        docker pull "$REGISTRY_HOST:$REGISTRY_PORT/$TARGET_IMAGE_NAME" \
+          && docker tag "$REGISTRY_HOST:$REGISTRY_PORT/$TARGET_IMAGE_NAME" "$TARGET_IMAGE_NAME" \
+          && docker rmi "$REGISTRY_HOST:$REGISTRY_PORT/$TARGET_IMAGE_NAME"
     done
-
-    # May need to find the service and remove it after use since Swarm will likely keep it alive
 }
 
 
 swarm__stop() {
     echo "Shutting down swarm..."
+    local carma_base=$(__get_image_from_config carma-base:)
```

Comment:
Should this be marked read-only? Also, the naming convention deviates from the rest of the script. Other variables use `SCREAMING_SNAKE_CASE`.